### ISSUE TITLE
eFolder: Display: 'block' prevents screen readers from reading data in table

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12119]
The names of files are displayed in a table visually with the column headers: Document Type, Source, and Receipt Date. This information is read as paragraph text for assistive technology users. This information should be coded as a table as it appears visually on the page. (New: 08/27/2019) the table is coded as a table however in Styles the display: block is checked, when I unchecked it JAWs read the table

**Comment by ajspotts** [comment]
Alisa I'm having trouble tracking down which view this issue is referring to.  Can you tell me where the view this is located at (the url) and which user type this pertains to.

**Comment by alisan16** [comment]
To reproduce this, you can load efolder locally and search for efolders: 561349913
eFolder Express is a Caseflow product that is in its own repo, so you'll need to follow these instructions to set it up: https://github.com/department-of-veterans-affairs/caseflow-efolder

**Comment by ajspotts** [comment]
Resolved with issue [#1130](https://github.com/department-of-veterans-affairs/caseflow-efolder/pull/1130)
